src/ServerConfigParams.ts

```diff
@@ -1,191 +1,197 @@
-    contractInteractor: ContractInteractor;
-    txStoreManager: TxStoreManager;
+  // TODO: rename as this name is terrible
+  managerKeyManager: KeyManager;
+  workersKeyManager: KeyManager;
+  contractInteractor: ContractInteractor;
+  txStoreManager: TxStoreManager;
 }
 
-const serverDefaultConfiguration: ServerConfigParams = {
-    alertedBlockDelay: 0,
-    minAlertedDelayMS: 0,
-    maxAlertedDelayMS: 0,
-    relayHubAddress: constants.ZERO_ADDRESS,
-    relayVerifierAddress: constants.ZERO_ADDRESS,
-    deployVerifierAddress: constants.ZERO_ADDRESS,
-    trustedVerifiers: [],
-    gasPriceFactor: 1,
-    registrationBlockRate: 0,
-    workerMinBalance: 0.001e18, // 0.001 RBTC
-    workerTargetBalance: 0.003e18, // 0.003 RBTC
-    managerMinBalance: 0.001e18, // 0.001 RBTC
-    managerMinStake: '1', // 1 wei
-    managerTargetBalance: 0.003e18, // 0.003 RBTC
-    minHubWithdrawalBalance: 0.001e18, // 0.001 RBTC
-    checkInterval: 10000,
-    readyTimeout: 30000,
-    devMode: false,
-    customReplenish: false,
-    logLevel: 1,
-    url: 'http://localhost:8090',
-    rskNodeUrl: '',
-    port: 0,
-    versionRegistryAddress: constants.ZERO_ADDRESS,
-    workdir: '',
-    refreshStateTimeoutBlocks: 5,
-    pendingTransactionTimeoutBlocks: 30, // around 5 minutes with 10 seconds block times
-    successfulRoundsForReady: 3, // successful mined blocks to become ready after exception
-    confirmationsNeeded: 12,
-    retryGasPriceFactor: 1.2,
-    defaultGasLimit: 500000,
-    maxGasPrice: (100e9).toString(),
-    estimateGasFactor: 1.2,
-    allowForSponsoredTx: true
+export const serverDefaultConfiguration: ServerConfigParams = {
+  // FIXME: serverDefaultConfiguration is not obvious and there is no other config. I suggest naming it just: defaultConfiguration
+  alertedBlockDelay: 0,
+  minAlertedDelayMS: 0,
+  maxAlertedDelayMS: 0,
+  relayHubAddress: constants.ZERO_ADDRESS,
+  relayVerifierAddress: constants.ZERO_ADDRESS,
+  deployVerifierAddress: constants.ZERO_ADDRESS,
+  trustedVerifiers: [],
+  gasPriceFactor: 1,
+  registrationBlockRate: 0,
+  workerMinBalance: 0.001e18, // 0.001 RBTC
+  workerTargetBalance: 0.003e18, // 0.003 RBTC
+  managerMinBalance: 0.001e18, // 0.001 RBTC
+  managerMinStake: '1', // 1 wei
+  managerTargetBalance: 0.003e18, // 0.003 RBTC
+  minHubWithdrawalBalance: 0.001e18, // 0.001 RBTC
+  checkInterval: 10000,
+  readyTimeout: 30000,
+  devMode: false,
+  customReplenish: false,
+  logLevel: 1,
+  url: 'http://localhost:8090',
+  rskNodeUrl: '',
+  port: 0,
+  versionRegistryAddress: constants.ZERO_ADDRESS,
+  workdir: '',
+  refreshStateTimeoutBlocks: 5,
+  pendingTransactionTimeoutBlocks: 30, // around 5 minutes with 10 seconds block times
+  successfulRoundsForReady: 3, // successful mined blocks to become ready after exception
+  confirmationsNeeded: 12,
+  retryGasPriceFactor: 1.2,
+  defaultGasLimit: 500000,
+  maxGasPrice: (100e9).toString(),
+  estimateGasFactor: 1.2,
+  workerFeePercentage: '0',
```

Comment:
if we change the param name, we need to change it here also

Reply:
done